content/metadata.yaml

```diff
@@ -434,11 +434,11 @@ authors:
     affiliations:
       - Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
       - Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
-    code of conduct:
+    code of conduct: yes
```

Comment:
```suggestion
    code of conduct:
```

Reply:
@anskelly I'm not sure if I did this (I am a little out of it this week) but it is causing a bit of an issue, would you be able to just accept this change? Sorry about this!!!

Reply:
@rando2 Yup no worries! I think I just accepted it! <3